src/lib/transaction_logic/parties_logic.ml

```diff
@@ -1078,6 +1078,11 @@ module Make (Inputs : Inputs_intf) = struct
     (* Set account timing for new accounts, if specified. *)
     let a, local_state =
       let timing = Party.Update.timing party in
+      let local_state =
+        Local_state.add_check local_state
+          Creation_of_timed_account_without_signature
```

Comment:
Can you use the existing `Update_not_permitted_timing_existing_account`, and remove that old check for `account_is_new`?